docs/app/partials/layout-tips.tmpl.html

```diff
@@ -108,98 +108,92 @@
   <br/>
   <hr/>
 
-  <h3>Layout Column and Container Heights</h3>
+  <h3 id="layout-column-0px-ie11">IE11 - Layout Column, 0px Height</h3>
 
   <p>
-    In Flexbox, some browsers will determine size of the flex containers based on the size of their
-    content. This is particularly noticable if you have a non-flex item (say a toolbar), followed by
-    two flex items in a column layout.
+    In Internet Explorer 11, when you have a column layout with unspecified height and flex items
+    inside, the browser can get confused and incorrectly calculate the height of each item (and thus
```

Comment:
#### Random English Silliness

I honestly think either would work here, but how I judge it is this: remove the "get confused and" phrase to see how it sounds:

> the browser can incorrectly calculate the height...

verses

> the browser can incorrectly calculates the height...

Honestly, seeing as I've never seen it **not** get confused, it might be more correct to just say

> the browser gets confused and incorrectly calculates the height...

Stupid english 😝